### PR TITLE
Feature/create product details page

### PR DESCRIPTION
**Pull request by maiorovvv**

### CODE REVIEW

File: src/features/ProductDetails/ProductDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import { useEffect } from 'react';
+
```
Comment (Igor-Klein-1988) [inline src/features/ProductDetails/ProductDetails.tsx]: delete empty line

File: src/features/ProductDetails/ProductDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import { useTranslation } from 'react-i18next';
+
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { RootState } from '../../app/store';
+
```
Comment (Igor-Klein-1988) [inline src/features/ProductDetails/ProductDetails.tsx]: delete empty line

File: src/features/ProductDetails/ProductDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import Spinner from '../../components/Spinner/Spinner';
+import { loadProduct } from './productDetailsSlice';
+
```
Comment (Igor-Klein-1988) [inline src/features/ProductDetails/ProductDetails.tsx]: delete empty line

File: src/features/ProductDetails/components/MediaSwiper.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}
+const MediaSwiper = (props: ProductProps): JSX.Element => {
+	const { product } = props;
+	const [thumbsSwiper, setThumbsSwiper] = useState(null);
+	const { t, i18n } = useTranslation('ProductDetails');
+
+	return (
+		// 					{product?.images?.length > 0 &&
+		// 				product.images.map((image, index) => (
+		// 					<SwiperSlide key={`swiper-${index}`}></SwiperSlide>
```
Comment (Igor-Klein-1988) [inline src/features/ProductDetails/components/MediaSwiper.tsx]: delete this

**Review by Igor-Klein-1988 — DISMISSED** [review]

File: src/features/products/Products.tsx [hunk 5]
```diff
@@ -63,7 +63,7 @@ const Products: React.FC = (): JSX.Element => {
 														<li className="product__items--action__list">
 															<div className="product__items--action__btn add__to--cart">
 																<Basket />
-																<span className="product_add__to--cart__text">
+																<span className="product_product__details_add__to--cart__text">
```
Comment (DenysBabenko29) [inline src/features/products/Products.tsx]: superfluous word (product)?

**Review by DenysBabenko29 — CHANGES_REQUESTED** [review]

**Review by DenysBabenko29 — APPROVED** [review]

**Pull request merged**